downstream/titles/release-notes/topics/aap-24.adoc

```diff
@@ -0,0 +1,111 @@
+// For each release of AAP, make a copy of this file and rename it to aap-rn-xx.adoc where xx is the release number; for example, 24 for the 2.4 release.
+// Save the renamed copy of this file to the release-notes/topics directory topic files for the release notes reside.
+//Only include release note types that have updates for a given release. For example, if there are no Technology previews for the release, remove that section from this file.
+
+
+= Ansible Automation Platform 2.4
+
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+
+== New features and enhancements
+
+This release of Ansible Automation Platform features the following enhancements:
+
+* Previously, the Execution Environment container images were based on RHEL 8 only. With Ansible Automation Platform 2.4 onwards, the Execution Environment container images are now available on RHEL 9 too. 
+The Execution Environment includes the following container images:
+** ansible-python-base
+** ansible-python-toolkit
+** ansible-builder
+** ee-minimal
+** ee-supported
+
+* The ansible-builder project recently released Ansible Builder version 3, a much-improved approach to creating execution environments. 
+You can use the following configuration YAML keys with Ansible Builder version 3:
+** additional_build_files
+** additional_build_steps
+** build_arg_defaults
+** dependencies
+** images
+** options
+** version
+For more information about using Ansible Builder version 3, see 
+https://ansible.readthedocs.io/projects/builder/en/stable/[Ansible Builder Documentation] and https://docs.ansible.com/automation-controller/latest/html/userguide/ee_reference.html[Execution Environment Setup Reference].
+
+* Ansible Automation Platform 2.4 and later versions can now be run on ARM platforms, including both the control plane and the execution environments.
+
+* Added an option to configure SSO logout URL for automation hub.
+
+* Updated the ansible-lint RPM package to version 6.14.3.
+
+* Updated django for potential denial-of-service vulnerability in file uploads (CVE-2023-24580).
+
+* Updated sqlparse for ReDOS vulnerability (CVE-2023-30608).
+
+* Updated django for potential denial-of-service via Accept-Language headers (CVE-2023-23969).
```

Comment:
Why is this one 'via' while line 40 is 'in'?

Reply:
Good point, changed it to 'in'